ui/src/app/components/gke-app-configuration-panels/create-gke-app.tsx

```diff
@@ -121,10 +132,38 @@ export const CreateGkeApp = ({
     autopauseThreshold: undefined,
   };
 
-  const createAppRequest: CreateAppRequest = {
-    ...defaultConfig,
-    persistentDiskRequest,
-  };
+  const [createAppRequest, setCreateAppRequest] =
+    React.useState<CreateAppRequest>({
+      ...defaultConfig,
+      persistentDiskRequest,
+      autodeleteEnabled:
+        app?.autodeleteEnabled ?? defaultConfig.autodeleteEnabled,
+      autodeleteThreshold:
+        app?.autodeleteThreshold ?? defaultConfig.autodeleteThreshold,
+    });
+
+  const autodeleteRemainingDays: number = (() => {
+    if (app?.autodeleteEnabled && app.dateAccessed && app.autodeleteThreshold) {
+      const dateAccessed = new Date(app.dateAccessed);
+      const autodeleteDate = new Date(
+        dateAccessed.getTime() + app.autodeleteThreshold * 60 * 1000
+      );
+
+      return getWholeDaysFromNow(autodeleteDate.getTime());
+    }
+    return null;
+  })();
+
+  const autodeleteToolTip = (
+    <div>
+      Deleting the environment when not in use will help reduce costs.
+      <br />
+      Reset Timer: Open the application to reset the timer
+      <br />
+      Disable Auto-Delete: Uncheck the box if you do not want your app to be
```

Comment:
this part is not strictly true **today**.  To disable auto-delete, it is necessary to manually delete the app, and re-create it with this unchecked.

This is a little confusing, so we'll need to work on the wording here.  Maybe: "disabling the auto-delete feature of a running environment is not currently possible."  Not sure if we want to say any more than that.

Reply:
Great, that works